guides-website/docs/typescript-style.md

```diff
@@ -554,3 +554,49 @@ function returnObject() {
 	};
 }
 ```
+
+## Import / Export
+
+- **Avoid export default**
+
+> We should avoid export default as the provide maintainability concerns. You can read more up about this topic in [Typescript Deep Dive - Avoid Export Default](https://basarat.gitbook.io/typescript/main-1/defaultisbad)
```

Comment:
```suggestion
> Avoid `export default` in TypeScript because of its poor maintainability and UX, as outlined in [Typescript Deep Dive - Avoid Export Default](https://basarat.gitbook.io/typescript/main-1/defaultisbad).
```